.github/workflows/runbook-freshness-check.yaml

```
@@ -0,0 +1,237 @@
+name: Runbook Freshness Check
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)